### ISSUE TITLE
After v0.19.0 release, problems with shapefiles

### ISSUE THREAD

**Issue by erogluorhan** [SciTools/cartopy#1772]
### Description
After `cartopy`'s version 0.19.0 release, shape file errors occurred in the continuous integration (CI) builds of some of our Python scripts. It was not the case before with v0.18.0 or older, and we are not able replicate this in our locals with the same conda list.

Here are two Python scripts and the errors we started to receive for them in Github actions:
- [NCL_polyg_2](https://github.com/NCAR/GeoCAT-examples/blob/main/Plots/Polygons/NCL_polyg_2.py): KeyError: "There is no item named 'ne_110m_admin_1_states_provinces_lakes_shp.cpg' in the archive" 
- [NCL_WRF_zoom_1_2](https://github.com/NCAR/GeoCAT-examples/blob/main/Plots/WRF/NCL_WRF_zoom_1_2.py): KeyError: "There is no item named 'ne_50m_admin_1_states_provinces_shp.cpg' in the archive"

Above errors can be found in our CI actions, e.g. [see this one](https://github.com/NCAR/GeoCAT-examples/runs/2442188579?check_suite_focus=true).

We tried downgrading `cartopy` to 0.18.0, and it seems to have worked. 

#### Code to reproduce

```
"""
NCL_polyg_2.py
==============
Concepts illustrated:
  - Drawing a Lambert Conformal U.S. map color-coded by climate divisions
  - Color-coding climate divisions based on precipitation values
  - Drawing the climate divisions of the U.S.
  - Zooming in on a particular area on a Lambert Conformal map
  - Drawing a border around filled polygons
  - Masking the ocean in a map plot
  - Masking land in a map plot
  - Increasing the font size of text
  - Adding text to a plot
  - Drawing a custom labelbar on a map
  - Creating a red-yellow-blue color map
See following URLs to see the reproduced NCL plot & script:
    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/polyg_2.ncl
    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/polyg_2_lg.png
"""

###############################################################################
# Import packages:

import cartopy.crs as ccrs
import cartopy.io.shapereader as shpreader
import geocat.datafiles as gdf
import matplotlib.cm as cm
import matplotlib.colors as colors
import matplotlib.pyplot as plt
import shapely.geometry as sgeom
import xarray as xr
from geocat.viz import util as gvutil
from mpl_toolkits.axes_grid1.inset_locator import inset_axes

###############################################################################
# Read in data

# Open climate division datafile and add to xarray
ds = xr.open_dataset(gdf.get("netcdf_files/climdiv_prcp_1899-1999.nc"),
                     decode_times=False)

###############################################################################
# Initialize color map and bounds for each color

colormap = colors.ListedColormap([
    'mediumpurple', 'mediumblue', 'royalblue', 'cornflowerblue', 'lightblue',
    'lightseagreen', 'yellowgreen', 'green', 'wheat', 'tan', 'gold', 'orange',
    'red', 'firebrick'
])

# Values represent average number of inches of rain
colorbounds = [0, 5, 10, 15, 20, 25, 30, 35, 40, 50, 60, 70, 80, 90, 100]

###############################################################################
# Define helper function to determine which color to fill the divisions based on precipitation data


def findDivColor(colorbounds, pdata):

    for x in range(len(colorbounds)):

        if pdata >= colorbounds[len(colorbounds) - 1]:
            return colormap.colors[x - 1]
        if pdata >= colorbounds[x]:
            continue
        else:
            # Index is 'x-1' because colorbounds is one item longer than colormap
            return colormap.colors[x - 1]


###############################################################################
# Create plot

# Create plot figure
fig = plt.figure(figsize=(8, 8))

# Add axes for lambert conformal map
# Set dimensions of axes with [X0, Y0, width, height] argument. Each value is a fraction of total figure size.
ax = plt.axes([.05, -.05, .9, 1],
              projection=ccrs.LambertConformal(),
              frameon=False)

# Set latitude and longitude extent of map
ax.set_extent([-119, -74, 18, 50], ccrs.Geodetic())

# Set shape name of map (which depicts the United States)
shapename = 'admin_1_states_provinces_lakes_shp'
states_shp = shpreader.natural_earth(resolution='110m',
                                     category='cultural',
                                     name=shapename)

# Set title and title fontsize of plot using gvutil function instead of matplotlib function call
gvutil.set_titles_and_labels(
    ax,
    maintitle=
    "Average Annual Precipiation \n Computed for the period 1899-1999 \n NCDC climate division data \n",
    maintitlefontsize=18)

# Add outlines of each state within the United States
for state in shpreader.Reader(states_shp).geometries():

    facecolor = 'white'
    edgecolor = 'black'

    ax.add_geometries([state],
                      ccrs.PlateCarree(),
                      facecolor=facecolor,
                      edgecolor=edgecolor)

# For each variable (climate division) in data set, create outline on map and fill with random color
for varname, da in ds.data_vars.items():

    # This condition is included because first item in xarray only has one attribute, 'current date'
    if hasattr(da, 'state_name'):
        # Get number of years of data by dividing number of months recorded (length of array) by 12 (12 months per year)
        numYears = len(da.values) / 12

        # Get precipitation data for each climate division:
        # Rather than looping through the whole array to find the sum of each 12 values (a year's worth of data),
        # adding each sum to an array, and then finding the average of the values in the array, as seen in the NCL
        # script, the one-line python method involves summing the dataset values and then dividing it by numYears (calculated in line 99)
        precipitationdata = sum(da.values) / numYears

        # Get borders of each climate division
        lat = da.lat
        lon = da.lon

        # Get color of climate division
        color = findDivColor(colorbounds, precipitationdata)

        # Use "shapely geometry" module to create division outlines from lat/lon coordinates
        track = sgeom.LineString(zip(lon, lat))

        # Add division outlines to map
        im = ax.add_geometries([track],
                               ccrs.PlateCarree(),
                               facecolor=color,
                               edgecolor='black',
                               linewidths=.5)

# Create and plot colorbar

# Map colors to bounds
norm = colors.BoundaryNorm(colorbounds, colormap.N)

# Add inset axes (axes within pre-existing axes) to hold colorbar
axins1 = inset_axes(ax, width="75%", height="3%", loc='lower center')

# Add colorbar to plot
cb = fig.colorbar(cm.ScalarMappable(cmap=colormap, norm=norm),
                  cax=axins1,
                  boundaries=colorbounds,
                  ticks=colorbounds,
                  spacing='uniform',
                  orientation='horizontal',
                  label='inches')

plt.show()
```

#### Traceback 

```
Running Sphinx v3.5.4
Downloading file 'registry.txt' from 'https://github.com/NCAR/GeoCAT-datafiles/raw/main/registry.txt' to '/Users/runner/Library/Caches/geocat'.
making output directory... done
WARNING: html_static_path entry '_static' does not exist
generating gallery...
Using Sphinx-Gallery to convert rst text blocks to markdown for .ipynb files.
generating gallery for gallery/Animations... [100%] NCL_animate_1.py
Downloading file 'netcdf_files/meccatemp.cdf' from 'https://github.com/NCAR/GeoCAT-datafiles/raw/main/netcdf_files/meccatemp.cdf' to '/Users/runner/Library/Caches/geocat'.
MovieWriter ffmpeg unavailable; using Pillow instead.

generating gallery for gallery/Bar... [ 25%] NCL_bar_11.py
generating gallery for gallery/Bar... [ 50%] NCL_bar_2.py
Downloading file 'netcdf_files/soi.nc' from 'https://github.com/NCAR/GeoCAT-datafiles/raw/main/netcdf_files/soi.nc' to '/Users/runner/Library/Caches/geocat'.
generating gallery for gallery/Bar... [ 75%] NCL_bar_7.py
generating gallery for gallery/Bar... [100%] NCL_unique_5.py

...

Extension error:
Here is a summary of the problems encountered when running the examples

Unexpected failing examples:
/Users/runner/work/GeoCAT-examples/GeoCAT-examples/Plots/WRF/NCL_WRF_zoom_1_2.py failed leaving traceback:
Traceback (most recent call last):
  File "/usr/local/miniconda/envs/geocat-examples/lib/python3.8/site-packages/sphinx_gallery/scrapers.py", line 333, in save_figures
    rst = scraper(block, block_vars, gallery_conf)
  File "/usr/local/miniconda/envs/geocat-examples/lib/python3.8/site-packages/sphinx_gallery/scrapers.py", line 149, in matplotlib_scraper
    fig.savefig(image_path, **these_kwargs)
  File "/usr/local/miniconda/envs/geocat-examples/lib/python3.8/site-packages/matplotlib/figure.py", line 2311, in savefig
    self.canvas.print_figure(fname, **kwargs)
  File "/usr/local/miniconda/envs/geocat-examples/lib/python3.8/site-packages/matplotlib/backend_bases.py", line 2210, in print_figure
    result = print_method(
  File "/usr/local/miniconda/envs/geocat-examples/lib/python3.8/site-packages/matplotlib/backend_bases.py", line 1639, in wrapper
    return func(*args, **kwargs)
  File "/usr/local/miniconda/envs/geocat-examples/lib/python3.8/site-packages/matplotlib/backends/backend_agg.py", line 509, in print_png
    FigureCanvasAgg.draw(self)
  File "/usr/local/miniconda/envs/geocat-examples/lib/python3.8/site-packages/matplotlib/backends/backend_agg.py", line 407, in draw
    self.figure.draw(self.renderer)
  File "/usr/local/miniconda/envs/geocat-examples/lib/python3.8/site-packages/matplotlib/artist.py", line 41, in draw_wrapper
    return draw(artist, renderer, *args, **kwargs)
  File "/usr/local/miniconda/envs/geocat-examples/lib/python3.8/site-packages/matplotlib/figure.py", line 1863, in draw
    mimage._draw_list_compositing_images(
  File "/usr/local/miniconda/envs/geocat-examples/lib/python3.8/site-packages/matplotlib/image.py", line 132, in _draw_list_compositing_images
    a.draw(renderer)
  File "/usr/local/miniconda/envs/geocat-examples/lib/python3.8/site-packages/matplotlib/artist.py", line 41, in draw_wrapper
    return draw(artist, renderer, *args, **kwargs)
  File "/usr/local/miniconda/envs/geocat-examples/lib/python3.8/site-packages/cartopy/mpl/geoaxes.py", line 517, in draw
    return matplotlib.axes.Axes.draw(self, renderer=renderer, **kwargs)
  File "/usr/local/miniconda/envs/geocat-examples/lib/python3.8/site-packages/matplotlib/artist.py", line 41, in draw_wrapper
    return draw(artist, renderer, *args, **kwargs)
  File "/usr/local/miniconda/envs/geocat-examples/lib/python3.8/site-packages/matplotlib/cbook/deprecation.py", line 411, in wrapper
    return func(*inner_args, **inner_kwargs)
  File "/usr/local/miniconda/envs/geocat-examples/lib/python3.8/site-packages/matplotlib/axes/_base.py", line 2748, in draw
    mimage._draw_list_compositing_images(renderer, self, artists)
  File "/usr/local/miniconda/envs/geocat-examples/lib/python3.8/site-packages/matplotlib/image.py", line 132, in _draw_list_compositing_images
    a.draw(renderer)
  File "/usr/local/miniconda/envs/geocat-examples/lib/python3.8/site-packages/matplotlib/artist.py", line 41, in draw_wrapper
    return draw(artist, renderer, *args, **kwargs)
  File "/usr/local/miniconda/envs/geocat-examples/lib/python3.8/site-packages/cartopy/mpl/feature_artist.py", line 153, in draw
    geoms = self._feature.intersecting_geometries(extent)
  File "/usr/local/miniconda/envs/geocat-examples/lib/python3.8/site-packages/cartopy/feature/__init__.py", line 297, in intersecting_geometries
    return super().intersecting_geometries(extent)
  File "/usr/local/miniconda/envs/geocat-examples/lib/python3.8/site-packages/cartopy/feature/__init__.py", line 106, in intersecting_geometries
    return (geom for geom in self.geometries() if
  File "/usr/local/miniconda/envs/geocat-examples/lib/python3.8/site-packages/cartopy/feature/__init__.py", line 279, in geometries
    path = shapereader.natural_earth(resolution=self.scale,
  File "/usr/local/miniconda/envs/geocat-examples/lib/python3.8/site-packages/cartopy/io/shapereader.py", line 282, in natural_earth
    return ne_downloader.path(format_dict)
  File "/usr/local/miniconda/envs/geocat-examples/lib/python3.8/site-packages/cartopy/io/__init__.py", line 203, in path
    result_path = self.acquire_resource(target_path, format_dict)
  File "/usr/local/miniconda/envs/geocat-examples/lib/python3.8/site-packages/cartopy/io/shapereader.py", line 344, in acquire_resource
    member = zfh.getinfo(member_path.replace(os.sep, '/'))
  File "/usr/local/miniconda/envs/geocat-examples/lib/python3.8/zipfile.py", line 1441, in getinfo
    raise KeyError(
KeyError: "There is no item named 'ne_50m_admin_1_states_provinces_shp.cpg' in the archive"


/Users/runner/work/GeoCAT-examples/GeoCAT-examples/Plots/Polygons/NCL_polyg_2.py failed leaving traceback:
Traceback (most recent call last):
  File "/Users/runner/work/GeoCAT-examples/GeoCAT-examples/Plots/Polygons/NCL_polyg_2.py", line 89, in <module>
    states_shp = shpreader.natural_earth(resolution='110m',
  File "/usr/local/miniconda/envs/geocat-examples/lib/python3.8/site-packages/cartopy/io/shapereader.py", line 282, in natural_earth
    return ne_downloader.path(format_dict)
  File "/usr/local/miniconda/envs/geocat-examples/lib/python3.8/site-packages/cartopy/io/__init__.py", line 203, in path
    result_path = self.acquire_resource(target_path, format_dict)
  File "/usr/local/miniconda/envs/geocat-examples/lib/python3.8/site-packages/cartopy/io/shapereader.py", line 344, in acquire_resource
    member = zfh.getinfo(member_path.replace(os.sep, '/'))
  File "/usr/local/miniconda/envs/geocat-examples/lib/python3.8/zipfile.py", line 1441, in getinfo
    raise KeyError(
KeyError: "There is no item named 'ne_110m_admin_1_states_provinces_lakes_shp.cpg' in the archive"


-------------------------------------------------------------------------------
make: *** [Makefile:20: html] Error 2
Error: Process completed with exit code 2.
```

### Operating system

Both `macos-latest` and `ubuntu-latest` OSs of Github Actions.

### Cartopy version

0.19.0

### conda list

```
# Name                    Version                   Build  Channel
alabaster                 0.7.12                     py_0    conda-forge
appdirs                   1.4.4              pyh9f0ad1d_0    conda-forge
appnope                   0.1.2            py38h50d1736_1    conda-forge
argon2-cffi               20.1.0           py38h5406a74_2    conda-forge
async_generator           1.10                       py_0    conda-forge
attrs                     20.3.0             pyhd3deb0d_0    conda-forge
babel                     2.9.0              pyhd3deb0d_0    conda-forge
backcall                  0.2.0              pyh9f0ad1d_0    conda-forge
backports                 1.0                        py_2    conda-forge
backports.functools_lru_cache 1.6.4              pyhd8ed1ab_0    conda-forge
bleach                    3.3.0              pyh44b312d_0    conda-forge
bokeh                     2.3.1            py38h50d1736_0    conda-forge
brotlipy                  0.7.0           py38h5406a74_1001    conda-forge
bzip2                     1.0.8                h0d85af4_4    conda-forge
c-ares                    1.17.1               h0d85af4_1    conda-forge
ca-certificates           2020.12.5            h033912b_0    conda-forge
cartopy                   0.19.0           py38h4be4431_0    conda-forge
cctools_osx-64            949.0.1             h6407bdd_21    conda-forge
certifi                   2020.12.5        py38h50d1736_1    conda-forge
cf_xarray                 0.5.1              pyh44b312d_0    conda-forge
cffi                      1.14.5           py38ha97d567_0    conda-forge
cfgv                      3.2.0                    pypi_0    pypi
cftime                    1.4.1            py38ha1b04c9_0    conda-forge
chardet                   4.0.0            py38h50d1736_1    conda-forge
clang                     11.1.0               h694c41f_0    conda-forge
clang-11                  11.1.0          default_he082bbe_0    conda-forge
clang_osx-64              11.1.0               hb91bd55_1    conda-forge
clangxx                   11.1.0          default_he082bbe_0    conda-forge
click                     7.1.2              pyh9f0ad1d_0    conda-forge
cloudpickle               1.6.0                      py_0    conda-forge
cmaps                     1.0.3              pyh9f0ad1d_0    conda-forge
colorama                  0.4.4              pyh9f0ad1d_0    conda-forge
compiler-rt               11.1.0               h654b07c_0    conda-forge
compiler-rt_osx-64        11.1.0               h8c5fa43_0    conda-forge
cryptography              3.4.7            py38h1fa4640_0    conda-forge
curl                      7.76.1               h06286d4_1    conda-forge
cycler                    0.10.0                     py_2    conda-forge
cytoolz                   0.11.0           py38h5406a74_3    conda-forge
dask                      2021.4.1           pyhd8ed1ab_0    conda-forge
dask-core                 2021.4.1           pyhd8ed1ab_0    conda-forge
dbus                      1.13.6               ha13b53f_2    conda-forge
decorator                 5.0.7              pyhd8ed1ab_0    conda-forge
defusedxml                0.7.1              pyhd8ed1ab_0    conda-forge
distlib                   0.3.1                    pypi_0    pypi
distributed               2021.4.1         py38h50d1736_0    conda-forge
docutils                  0.16             py38h50d1736_3    conda-forge
entrypoints               0.3             pyhd8ed1ab_1003    conda-forge
eofs                      1.4.0                      py_0    conda-forge
expat                     2.3.0                he49afe7_0    conda-forge
filelock                  3.0.12                   pypi_0    pypi
freetype                  2.10.4               h4cff582_1    conda-forge
fsspec                    2021.4.0           pyhd8ed1ab_0    conda-forge
geocat-comp               2021.04.0                  py_0    ncar
geocat-datafiles          2021.1.0                   py_0    ncar
geocat-f2py               2021.04.0        py38hae5c3c8_0    ncar
geocat-viz                2020.7.30.1        pyh39e3cac_0    ncar
geographiclib             1.50                       py_0    conda-forge
geos                      3.9.1                he49afe7_2    conda-forge
gettext                   0.19.8.1          h7937167_1005    conda-forge
gfortran_impl_osx-64      9.3.0               h9cc0e5e_22    conda-forge
gfortran_osx-64           9.3.0               h18f7dce_14    conda-forge
glib                      2.68.1               he49afe7_0    conda-forge
glib-tools                2.68.1               he49afe7_0    conda-forge
gmp                       6.2.1                h2e338ed_0    conda-forge
hdf4                      4.2.13            hefd3b78_1005    conda-forge
hdf5                      1.10.6          nompi_hc5d9132_1114    conda-forge
heapdict                  1.0.1                      py_0    conda-forge
icu                       68.1                 h74dc148_0    conda-forge
identify                  2.2.4                    pypi_0    pypi
idna                      2.10               pyh9f0ad1d_0    conda-forge
imagesize                 1.2.0                      py_0    conda-forge
importlib-metadata        4.0.1            py38h50d1736_0    conda-forge
importlib_metadata        4.0.1                hd8ed1ab_0    conda-forge
importlib_resources       5.1.2            py38h50d1736_0    conda-forge
ipykernel                 5.5.3            py38h6c79ece_0    conda-forge
ipython                   7.22.0           py38h6c79ece_0    conda-forge
ipython_genutils          0.2.0                      py_1    conda-forge
ipywidgets                7.6.3              pyhd3deb0d_0    conda-forge
isl                       0.22.1               hb1e8313_2    conda-forge
jedi                      0.18.0           py38h50d1736_2    conda-forge
jinja2                    2.11.3             pyh44b312d_0    conda-forge
joblib                    1.0.1              pyhd8ed1ab_0    conda-forge
jpeg                      9d                   hbcb3906_0    conda-forge
jsonschema                3.2.0              pyhd8ed1ab_3    conda-forge
jupyter                   1.0.0            py38h50d1736_6    conda-forge
jupyter_client            6.1.12             pyhd8ed1ab_0    conda-forge
jupyter_console           6.4.0              pyhd8ed1ab_0    conda-forge
jupyter_core              4.7.1            py38h50d1736_0    conda-forge
jupyterlab_pygments       0.1.2              pyh9f0ad1d_0    conda-forge
jupyterlab_widgets        1.0.0              pyhd8ed1ab_1    conda-forge
kiwisolver                1.3.1            py38hd9c93a9_1    conda-forge
krb5                      1.17.2               h60d9502_0    conda-forge
lcms2                     2.12                 h577c468_0    conda-forge
ld64_osx-64               530                 he8994da_21    conda-forge
ldid                      2.1.2                h7660a38_2    conda-forge
libblas                   3.9.0                8_openblas    conda-forge
libcblas                  3.9.0                8_openblas    conda-forge
libclang                  11.1.0          default_he082bbe_0    conda-forge
libclang-cpp11.1          11.1.0          default_he082bbe_0    conda-forge
libcurl                   7.76.1               h8ef9fac_1    conda-forge
libcxx                    11.1.0               habf9029_0    conda-forge
libedit                   3.1.20191231         h0678c8f_2    conda-forge
libev                     4.33                 haf1e3a3_1    conda-forge
libffi                    3.3                  h046ec9c_2    conda-forge
libgfortran               5.0.0           9_3_0_h6c81a4c_22    conda-forge
libgfortran-devel_osx-64  9.3.0               h6c81a4c_22    conda-forge
libgfortran5              9.3.0               h6c81a4c_22    conda-forge
libglib                   2.68.1               hd556434_0    conda-forge
libiconv                  1.16                 haf1e3a3_0    conda-forge
liblapack                 3.9.0                8_openblas    conda-forge
libllvm11                 11.1.0               hd011deb_2    conda-forge
libnetcdf                 4.8.0           nompi_h81fa352_101    conda-forge
libnghttp2                1.43.0               h07e645a_0    conda-forge
libopenblas               0.3.12          openmp_h54245bb_1    conda-forge
libpng                    1.6.37               h7cec526_2    conda-forge
libpq                     13.2                 h052a64a_2    conda-forge
libsodium                 1.0.18               hbcb3906_1    conda-forge
libssh2                   1.9.0                h52ee1ee_6    conda-forge
libtiff                   4.2.0                h7c11950_1    conda-forge
libwebp-base              1.2.0                h0d85af4_2    conda-forge
libzip                    1.7.3                hbc046b2_0    conda-forge
llvm-openmp               11.1.0               hda6cdc1_1    conda-forge
llvm-tools                11.1.0               hd011deb_2    conda-forge
locket                    0.2.0                      py_2    conda-forge
lz4-c                     1.9.3                h046ec9c_0    conda-forge
make                      4.3                  h22f3db7_1    conda-forge
markupsafe                1.1.1            py38h5406a74_3    conda-forge
matplotlib                3.3.0                         1    conda-forge
matplotlib-base           3.3.0            py38haed9462_1    conda-forge
metpy                     1.0                pyhd8ed1ab_0    conda-forge
mistune                   0.8.4           py38h5406a74_1003    conda-forge
mock                      4.0.3            py38h50d1736_1    conda-forge
mpc                       1.1.0             ha57cd0f_1009    conda-forge
mpfr                      4.0.2                h72d8aaf_1    conda-forge
msgpack-python            1.0.2            py38hd9c93a9_1    conda-forge
mysql-common              8.0.23               h694c41f_1    conda-forge
mysql-libs                8.0.23               hbeb7981_1    conda-forge
nbclient                  0.5.3              pyhd8ed1ab_0    conda-forge
nbconvert                 6.0.7            py38h50d1736_3    conda-forge
nbformat                  5.1.3              pyhd8ed1ab_0    conda-forge
ncurses                   6.2                  h2e338ed_4    conda-forge
nest-asyncio              1.5.1              pyhd8ed1ab_0    conda-forge
netcdf4                   1.5.6           nompi_py38h8ce633d_103    conda-forge
nodeenv                   1.6.0                    pypi_0    pypi
notebook                  6.3.0              pyha770c72_1    conda-forge
nspr                      4.30                 hcd9eead_0    conda-forge
nss                       3.64                 h31e2bf1_0    conda-forge
numpy                     1.19.5           py38h6ced74f_1    conda-forge
olefile                   0.46               pyh9f0ad1d_1    conda-forge
openjpeg                  2.4.0                h6cbf5cd_0    conda-forge
openssl                   1.1.1k               h0d85af4_0    conda-forge
packaging                 20.9               pyh44b312d_0    conda-forge
pandas                    1.2.4            py38h1f261ad_0    conda-forge
pandoc                    2.13                 h0d85af4_0    conda-forge
pandocfilters             1.4.2                      py_1    conda-forge
parso                     0.8.2              pyhd8ed1ab_0    conda-forge
partd                     1.2.0              pyhd8ed1ab_0    conda-forge
pcre                      8.44                 hb1e8313_0    conda-forge
pexpect                   4.8.0              pyh9f0ad1d_2    conda-forge
pickleshare               0.7.5                   py_1003    conda-forge
pillow                    8.1.2            py38h83525de_1    conda-forge
pint                      0.17               pyhd8ed1ab_0    conda-forge
pip                       21.1               pyhd8ed1ab_0    conda-forge
pooch                     1.3.0              pyhd8ed1ab_0    conda-forge
pre-commit                2.12.1                   pypi_0    pypi
proj                      7.2.0                h78d1473_2    conda-forge
prometheus_client         0.10.1             pyhd8ed1ab_0    conda-forge
prompt-toolkit            3.0.18             pyha770c72_0    conda-forge
prompt_toolkit            3.0.18               hd8ed1ab_0    conda-forge
psutil                    5.8.0            py38h96a0964_1    conda-forge
ptyprocess                0.7.0              pyhd3deb0d_0    conda-forge
pycparser                 2.20               pyh9f0ad1d_2    conda-forge
pygments                  2.8.1              pyhd8ed1ab_0    conda-forge
pyopenssl                 20.0.1             pyhd8ed1ab_0    conda-forge
pyparsing                 2.4.7              pyh9f0ad1d_0    conda-forge
pyproj                    3.0.1            py38hbfd3961_0    conda-forge
pyqt                      5.12.3           py38h50d1736_7    conda-forge
pyqt-impl                 5.12.3           py38h721a93c_7    conda-forge
pyqt5-sip                 4.19.18          py38h5745d40_7    conda-forge
pyqtchart                 5.12             py38h721a93c_7    conda-forge
pyqtwebengine             5.12.1           py38h721a93c_7    conda-forge
pyrsistent                0.17.3           py38h5406a74_2    conda-forge
pyshp                     2.1.3              pyh44b312d_0    conda-forge
pysocks                   1.7.1            py38h50d1736_3    conda-forge
python                    3.8.8           h4e93d89_0_cpython    conda-forge
python-dateutil           2.8.1                      py_0    conda-forge
python_abi                3.8                      1_cp38    conda-forge
pytz                      2021.1             pyhd8ed1ab_0    conda-forge
pyyaml                    5.4.1            py38h5406a74_0    conda-forge
pyzmq                     22.0.3           py38hd3b92b6_1    conda-forge
qt                        5.12.9               h126340a_4    conda-forge
qtconsole                 5.0.3              pyhd8ed1ab_0    conda-forge
qtpy                      1.9.0                      py_0    conda-forge
readline                  8.1                  h05e3726_0    conda-forge
requests                  2.25.1             pyhd3deb0d_0    conda-forge
scikit-learn              0.24.1           py38hfd19401_0    conda-forge
scipy                     1.6.2            py38h431c0a8_0    conda-forge
send2trash                1.5.0                      py_0    conda-forge
setuptools                49.6.0           py38h50d1736_3    conda-forge
shapely                   1.7.1            py38h08f9a40_4    conda-forge
six                       1.15.0             pyh9f0ad1d_0    conda-forge
snowballstemmer           2.1.0              pyhd8ed1ab_0    conda-forge
sortedcontainers          2.3.0              pyhd8ed1ab_0    conda-forge
sphinx                    3.5.4              pyh44b312d_0    conda-forge
sphinx-gallery            0.9.0              pyhd8ed1ab_0    conda-forge
sphinx_rtd_theme          0.5.2              pyhd8ed1ab_1    conda-forge
sphinxcontrib-applehelp   1.0.2                      py_0    conda-forge
sphinxcontrib-devhelp     1.0.2                      py_0    conda-forge
sphinxcontrib-htmlhelp    1.0.3                      py_0    conda-forge
sphinxcontrib-qthelp      1.0.3                      py_0    conda-forge
sphinxcontrib-serializinghtml 1.1.4                      py_0    conda-forge
sqlite                    3.35.5               h44b9ce1_0    conda-forge
tapi                      1100.0.11            h9ce4665_0    conda-forge
tblib                     1.7.0              pyhd8ed1ab_0    conda-forge
terminado                 0.9.4            py38h50d1736_0    conda-forge
testpath                  0.4.4                      py_0    conda-forge
threadpoolctl             2.1.0              pyh5ca1d4c_0    conda-forge
tk                        8.6.10               h0419947_1    conda-forge
toml                      0.10.2                   pypi_0    pypi
toolz                     0.11.1                     py_0    conda-forge
tornado                   6.1              py38h5406a74_1    conda-forge
traitlets                 5.0.5                      py_0    conda-forge
typing_extensions         3.7.4.3                    py_0    conda-forge
urllib3                   1.26.4             pyhd8ed1ab_0    conda-forge
virtualenv                20.4.4                   pypi_0    pypi
wcwidth                   0.2.5              pyh9f0ad1d_2    conda-forge
webencodings              0.5.1                      py_1    conda-forge
wheel                     0.36.2             pyhd3deb0d_0    conda-forge
widgetsnbextension        3.5.1            py38h50d1736_4    conda-forge
wrapt                     1.12.1           py38h5406a74_3    conda-forge
wrf-python                1.3.2.5          py38h0ffacb2_0    conda-forge
xarray                    0.17.0             pyhd8ed1ab_0    conda-forge
xz                        5.2.5                haf1e3a3_1    conda-forge
yaml                      0.2.5                haf1e3a3_0    conda-forge
zeromq                    4.3.4                h1c7c35f_0    conda-forge
zict                      2.0.0                      py_0    conda-forge
zipp                      3.4.1              pyhd8ed1ab_0    conda-forge
zlib                      1.2.11            h7795811_1010    conda-forge
zstd                      1.4.9                h582d3a0_0    conda-forge
```

**Comment by greglucas** [comment]
Geocat must be archiving a stripped down version of the NaturalEarth files?

If I download that feature directly from NaturalEarth I get the `.cpg` file: https://www.naturalearthdata.com/downloads/50m-cultural-vectors/

Can you update the contents of that archive, or download directly from NaturalEarth using Cartopy's routines?

This was added to the read routines in #1745.

**Comment by erogluorhan** [comment]
Thanks @greglucas !

No, we are not archiving the NaturalEarth files, but I now figured out the actual issue.

I realized we were giving the shape file names with "_shp" at the end of each of them (I don't know why). For example, "admin_1_states_provinces_lakes_shp" and "admin_1_states_provinces_shp". However, this naming was working until `cartopy` 0.19.0. But with 0.19.0, it apparently could not files with a name "... _shp.cpg". 

I have removed the "_shp" parts from the names, and they are working now.
